app/src/main/java/com/android/solvit/seeker/ui/provider/ProviderInfo.kt

```diff
@@ -0,0 +1,401 @@
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import coil.compose.AsyncImage
+import com.android.solvit.R
+import com.android.solvit.seeker.model.provider.ListProviderViewModel
+import com.android.solvit.shared.model.provider.Provider
+import com.android.solvit.shared.model.review.Review
+import com.android.solvit.shared.model.review.ReviewViewModel
+import com.android.solvit.shared.ui.navigation.NavigationActions
+
+@Composable
+fun ProviderInfoScreen(
+    navigationActions: NavigationActions,
+    providerViewModel: ListProviderViewModel = viewModel(factory = ListProviderViewModel.Factory),
+    reviewsViewModel: ReviewViewModel = viewModel(factory = ReviewViewModel.Factory)
+) {
+  val provider = providerViewModel.selectedProvider.collectAsState().value ?: return
+  val reviews =
+      reviewsViewModel.reviews.collectAsState().value.filter { it.providerId == provider.uid }
+
+  var selectedTabIndex by remember { mutableIntStateOf(0) }
+
+  Scaffold(
+      topBar = { ProviderTopBar(onBackClick = { navigationActions.goBack() }) },
+      content = { padding ->
+        Column(modifier = Modifier.padding(padding)) {
+          ProviderHeader(provider)
+          ProviderTabs(selectedTabIndex) { newIndex -> selectedTabIndex = newIndex }
+
+          // Display content based on the selected tab
+          when (selectedTabIndex) {
+            0 ->
+                ProviderDetails(
+                    provider, reviews) // Display ProviderDetails if "Profile" tab is selected
+            1 ->
+                ProviderReviews(
+                    provider, reviews) // Display ProviderReviews if "Reviews" tab is selected
+          }
+        }
+      },
+      bottomBar = { BottomBar() })
+}
+
+@Composable
+fun ProviderTopBar(onBackClick: () -> Unit) {
+  val context = LocalContext.current
+  Row(
+      modifier = Modifier.fillMaxWidth().background(color = Color.White).testTag("ProviderTopBar"),
+      verticalAlignment = Alignment.CenterVertically) {
+        // Back button on the left
+        IconButton(onClick = onBackClick, modifier = Modifier.testTag("backButton")) {
+          Icon(Icons.AutoMirrored.Filled.ArrowBack, contentDescription = "Back")
+        }
+
+        // Title in the center
+        Text(
+            text = "Performer",
```

Comment:
Nit: Shouldn't it be "Provider" instead of "Performer"? It could maybe even include the mention of the Provider's name.

Reply:
I simply copied the Figma. I modified this field according to your advice.